ng-dev/auth/shared/ng-dev-token.ts

```diff
@@ -65,51 +88,48 @@ export async function requestNgDevToken() {
   const {data: token} = await invokeServerFunctionUnsafe<{}, string>('ngDevTokenRequest', {
     idToken: await auth.currentUser.getIdToken(),
   });
-  ngDevToken = {token, email: auth.currentUser.email || 'unknown email'};
-  await saveTokenToFileSystem(ngDevToken);
+  ngDevUserToken = {token, email: auth.currentUser.email || 'unknown email'};
+  await saveTokenToFileSystem(ngDevUserToken);
+  return getUserWithoutToken(ngDevUserToken);
 }
```

Comment:
Maybe store the object without token directly and just cache it? Seems low effort and allows us to just use it as a constant